src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,7 +207,7 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().startsWith(resource.getId())) {
+            if (trigger.getId().endsWith(resource.getId())) {
```

Comment:
Actually that will force to Do a bigger part of code change. As at present code is not saving Tag part. Will do that later. In individual pull request. Here It was mainly to cure alert part.